torch/_functorch/config.py

```diff
@@ -205,6 +205,10 @@ def remote_autograd_cache_default() -> Optional[bool]:
 # Used for tests
 strict_autograd_cache = False
 
+# See Note [AOTAutograd Tangent Subclassness for mutated inputs]
+# TODO(ivankobzarev): Remove this config, being able to deduce it compile time.
```

Comment:
idk if we're ever actually going to be able to deduce this at compile time, unless we actually commit to the "don't specialize tangents until backward runs" work :/